storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -21,6 +21,12 @@ Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperabilit
 
 This feature is *tech preview* quality.
 
+Percona XtraBackup 8.0.28-21 adds support for the Amazon Key Management Service (AWS KMS). AWS KMS is an encryption and key management service used in the cloud. The keys and functionality can be used for other AWS services or your applications that use AWS. 
```

Comment:
Can we fix all three of the references?
Percona XtraBackup 8.0.25-17  adds support for the keyring_file component, which is part of the component-based infrastructure MySQL which extends the server capabilities. The component is stored in the plugin directory.
Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server. This feature is tech preview quality.
Percona XtraBackup 8.0.28-21 adds support for the Amazon Key Management Service (AWS KMS). AWS KMS is a cloud-based encryption and key management service. The keys and functionality can be used for other AWS services or your applications that use AWS. This feature is tech preview quality. (edited)